task_1.py

```diff
@@ -0,0 +1,10 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+
+
+# TODO заменить значение пропущенного элемента средним арифметическим
+incorrect_none = numbers[4]
+correct_none = 0
```

Comment:
Чтобы не вводить лишние переменные, вы могли сразу присвоить значение 0 
```suggestion
numbers[4] = 0
```